docs/user-management.md

```diff
@@ -0,0 +1,41 @@
+# User management
+
+With the GitHub API, you can manage your own user account.
+
+Theses details include the following:
```

Comment:
Theses > These

Reply:
Well, that is a slightly embarrassing typo on my behalf.